### ISSUE TITLE
Invalid Captcha Token While Creating account from admin

### ISSUE THREAD

**Issue by surajTupeBorn** [gigya/Magento-2#96]
Hello there,

I'm trying to create an account from backed, but getting the below error while submitting the details.

API Method :  apiMethod=accounts.register
URL: https://accounts.us1.gigya.com/accounts.register

`Gigya\PHP\GSResponse Object
(
    [errorCode:Gigya\PHP\GSResponse:private] => 400009
    [errorMessage:Gigya\PHP\GSResponse:private] => Validation error
    [rawData:Gigya\PHP\GSResponse:private] => {
  "callId": "2dae999a964e463abd8d939ad4bd835a",
  "errorCode": 400009,
  "errorDetails": "Validation failed",
  "errorMessage": "Validation error",
  "apiVersion": 2,
  "statusCode": 400,
  "statusReason": "Bad Request",
  "time": "2020-11-24T08:21:07.772Z",
  "validationErrors": [
    {
      "errorCode": 400021,
      "message": "",
      "fieldName": "captchaToken"
    }
  ]
}
    [data:Gigya\PHP\GSResponse:private] => Gigya\PHP\GSObject Object
        (
            [map:Gigya\PHP\GSObject:private] => Array
                (
                    [callId] => 2dae999a964e463abd8d939ad4bd835a
                    [errorCode] => 400009
                    [errorDetails] => Validation failed
                    [errorMessage] => Validation error
                    [apiVersion] => 2
                    [statusCode] => 400
                    [statusReason] => Bad Request
                    [time] => 2020-11-24T08:21:07.772Z
                    [validationErrors] => Gigya\PHP\GSArray Object
                        (
                            [map:Gigya\PHP\GSArray:private] => Array
                                (
                                    [0] => Gigya\PHP\GSObject Object
                                        (
                                            [map:Gigya\PHP\GSObject:private] => Array
                                                (
                                                    [errorCode] => 400021
                                                    [message] => 
                                                    [fieldName] => captchaToken
                                                )

                                        )

                                )

                        )

                )

        )
`
NOTE : 1) Captcha is enabled at the GIGYA console.
             2) Frontend registration is working correctly.

Thanks in advance!!

**Comment by levistepanov** [comment]
I might be wrong, but i think captcha is only supported from client-side. why are you trying to use it server side... you should trust your server is who it says it is.

**Comment by surajTupeBorn** [comment]
@levistepanov Correct, I'm not sure but the Invisible Captcha is managed on Global Level, that is why it is giving "Validation error" on "fieldName": "captchaToken". 
After disabled Captch, It is working fine.

![Screenshot - 2020-11-24T160212 208](https://user-images.githubusercontent.com/69463321/100082811-b8677400-2e6e-11eb-8b0a-43ced5bcedde.png)
![Screenshot - 2020-11-24T160127 398](https://user-images.githubusercontent.com/69463321/100082828-be5d5500-2e6e-11eb-83c8-be2f2f9e9685.png)

Is there any parameter to disable captcha while submitting a request to the server throgh API?

**Comment by levistepanov** [comment]
Only thing i can think of is, maybe you are not passing credentials in the server request? if you do not pass key/secret, it will consider it a client-side call and require captcha. try adding your appkey and secret to the request (if not already doing so) and see if it helps. If that isn't it, will have to wait for ynhockey to reply.

$request = new GSRequest($apiKey,$secretKey,$method, null, true, $userKey);

**Comment by surajTupeBorn** [comment]
@levistepanov, I'm using a default API call, it includes everything $apiKey,$secretKey,$method, null, true, and $userKey.

**Comment by surajTupeBorn** [comment]
Just follow up: @Ynhockey @levistepanov any resolution or guidance?

**Comment by Ynhockey** [comment]
Hi Suraj,

Sorry for the late reply. I will look to see if there is a solution for you, generally you need to send a CAPTCHA token with accounts.register when the policy is on, which the Magento module isn't built for.

However, please open a ticket with support so that we can track the issue internally and respond accordingly.
